vta/hardware/chisel/Makefile

```diff
@@ -32,16 +32,19 @@ ifeq (, $(VERILATOR_INC_DIR))
   endif
 endif
 
-CONFIG = DefaultPynqConfig
+CONFIG = DefaultDe10Config
```

Comment:
If we're going to set the default to DE10, has the PynqConfig been tested in simulation as well in this PR?

Reply:
I was not aware that pynq was also used as a target for Chisel.
Have not tested in simulation, so I would use De10 as the working one.